### DISCUSSION TITLE
你们是看不到用户发的帖子吗！！！！！！！！！

### DISCUSSION THREAD

**Discussion by liuh23** [getlantern/lantern-forum-cn#1014] [General]
反馈了很多次，怎么一直不回应！！！！！！

我的邮箱mrmaketo1@gmail.com
我的回复码：78232230
我之前讨论的帖子https://github.com/getlantern/lantern-forum-cn/discussions/973

**Comment by liuh23** [comment]
![380472046-701487c9-17ac-4797-bf4e-a82583efe419](https://github.com/user-attachments/assets/b20aea59-546f-4abc-b544-896839f38428)

**Comment by MaxMKMax** [comment]
@liuh23 您好. 已经帮您提交好退款申请了. 我再帮您提醒一下. 谢谢您的耐心等待!

**Comment by liuh23** [comment]
多久能到账？

**Comment by liuh23** [comment]
三周过去了，你已经提交过退款申请，而且还提醒了一次。退款的钱呢？你就只是说说而已吗？？

**Comment by MaxMKMax** [comment]
@liuh23 您好. 我也不懂情况了, 因为不是我处理退款的. 我已经帮您再次询问了. 不好意思啊.
我们一般都会给用户退款的, 只要第三方支付系统没问题. 或者您可以试试看在信用卡或支付宝取消消费还是申请退款看可以吗?

**Comment by greySkiesMarsh** [comment]
Hello, I'm very sorry for the delay in resolving this issue. 

Your refund request has been approved, but this payment processor requires refunds to be handled internally, so it's outside our ability to process this ourselves. Please file a dispute with the payment processor listed on your receipt and they will handle it from there.

**Comment by liuh23** [comment]
退款完全就是骗人的，你申请退款时会告诉你已经向上提交。
如果你多次催促，会让你去支付宝申请退款。
申请之后，你得到的商家回复是：”账户限制，无法退款，你需要跟支付宝客服申请退款“。
找了客服申请后，再次得到的商家回复是：”账户被限制了，要等180天账户解除限制才能退款“。
-------------------总之就是各种理由不给你退，一直拖着，照片为证。（如果你们诚心退款，那就直接支付宝，或者微信转账给我啊）

**Reply by liuh23** [reply]
![image](https://github.com/user-attachments/assets/a8d89ddd-237e-4d29-a887-b2a0e358c033)

**Comment by liuh23** [comment]
支付宝和这个论坛，都让你联系这个收款方：广州饶铭网络科技有限公司。
但是它根本没有预留电话，你是联系不到它的。再者说，它显然是一家国内广州的公司，怎么会需要”向上提交问题才能退款“
你们明明是同一家公司，结果让我去联系它，开玩笑吗？

**Reply by Ginny125** [reply]
我也是，支付宝账单申请退款并投诉，显示已处理但是没收到退款

**Reply by Ginny125** [reply]
而且蓝灯也变免费版了，这个处理的倒是快，密码的

**Comment by MaxMKMax** [comment]
@liuh23 您试试看我们支付部门客服的建议先好不好.? 因为海外和国内的系统不一样. 我们这里的系统弄不了.

**Comment by liuh23** [comment]
我就是因为试过了没用才生气啊。跟支付宝的客服，申请退款，然后你们商家给的回复是，账号限制，让我等180天解除限制

**Reply by MaxMKMax** [reply]
@liuh23 您可以提供我那个回复的邮件吗? 邮件的贴图吗? 我帮您找支付团队问看. 不好意思啊, 非常抱歉. 希望您息怒. 真的万分抱歉!

**Comment by liuh23** [comment]
我现在找一下，你先别下线

**Reply by liuh23** [reply]
![image](https://github.com/user-attachments/assets/09eb80d3-fbc8-4369-8577-8d2beefa44f5)

**Comment by liuh23** [comment]
又是一周过去了，你看的怎么样了，又没下文了？？？？

**Comment by liuh23** [comment]
我的标题就是为了让你看到帖子，你还偷偷改我标题，过分了吧。
你解决了问题，我自然会删贴的

**Comment by MaxMKMax** [comment]
@liuh23 您好. 但问题是我们的负责人已经和您说清楚我们这里没权限处理您的退款.  需要您向第三方那里 / 支付宝申请退款.

有话好好说. 你那么激动干嘛. 我都说了会尽力帮您. 你激动反而更难搞.

![image](https://github.com/user-attachments/assets/36b5e98b-8cc9-439e-9b03-af928189c986)

**Reply by liuh23** [reply]
可是我申请退款过后，收款方的回复是：账号有问题，退不了。
那我等你两个星期，12月24号能帮我处理好吗（从我申请退款到现在已经快两个月了）。
支付宝退不了的话，我可以把银行卡发你邮箱，用银行卡退

**Reply by seagull2024** [reply]
看了这么多，原来大家都一样

**Comment by MaxMKMax** [comment]
![image](https://github.com/user-attachments/assets/775d3334-d6ba-4d28-b6e4-d716c88d80ee)
@liuh23 您好. 之前也给您答复了. 我们这里需要您和第三方提交退款. 我们是交给第三方公式处理的. 非常抱歉啊!
我们自己的系统上是无法弄退款的. 我们的技术人员也给您说过了. 我们这里已经批准退款了, 但还需要您和支付宝或这个第三方的公式弄退款.

**Comment by MaxMKMax** [comment]
@liuh23 
支付部门的翻译:
您的退款请求已获批准，但此付款处理商要求退款在内部处理，因此我们无法自行处理。请向收据上列出的付款处理商提出争议，他们会处理。
